apps/dashboard/config/locales/en.yml

```diff
@@ -178,6 +178,7 @@ en:
     jobs_project_not_found: Cannot find project %{project_id}
     jobs_project_save_error: Cannot save manifest to %{path}
     jobs_project_validation_error: Invalid Request. Please review the errors below
+    jobs_select_directory_placeholder: List of importable directories based upon your group-id
```

Comment:
These are not based on group IDs.

Reply:
I will keep it short - "List of importable directories"